website/ref/language.md

```diff
@@ -106,16 +106,75 @@ error.
 
 The three syntaxes above all evaluate to strings, and they are interchangeable.
 For instance, `xyz`, `'xyz'` and `"xyz"` are different syntaxes for the same
-string, and they are always equivalent.
+string, and they are always equivalent with the exception of
+**escape sequences** as documented above.
 
-Elvish does have a dedicated `float64` number type, but it does not have a
-literal syntax; instead it needs to be constructed using a function call, as in
-`put (float64 42)`; the argument `42` is a string. For convenience, all Elvish
-functions that expect number arguments also accept strings that can be parsed as
-a number, so explicit number constructors are not needed often.
+# Numeric data type
 
+Elvish has a numeric data type, `float64`, created using this syntax:
+`(float64 numeric-literal)`. For convenience, all Elvish functions that
+expect a numeric argument will accept a string that can be parsed as a
+number. They will automatically convert the string to a `float64`. Which
+means explicit `float64` constructors are almost never needed. Not even
+for special values like infinity (`Inf`) or not-a-number (`NaN`). Note
+that the letter case of these magic values doesn't matter so `inf` and
+`Inf` are equivalent.
 
-# List and Map
+Numeric literals can be specified using
```

Comment:
See above; these are not literals. If you used the plaeceholder `$number` above, just say:

The $number argument can be a string in one of the following formats: